src/gwc/src/main/java/org/geoserver/gwc/controller/GwcRestWebUrlHandlerMapping.java

```diff
@@ -0,0 +1,19 @@
+/* (c) 2022 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.gwc.controller;
+
+import org.geoserver.catalog.Catalog;
+
+/**
+ * Handler for mapping workspace-based web (i.e. web/openlayers/ol.js) requests to
+ * non-workspace-based requests.
+ */
+public class GwcRestWebUrlHandlerMapping extends GwcWmtsRestUrlHandlerMapping {
```

Comment:
Unsure about this one... is this controller doing more or less the same thing as the "www" folder in GeoServer? If so maybe it should stay in gwc-core.

I don't see documentation for this endpoint:
* https://docs.geoserver.org/stable/en/user/geowebcache/rest/index.html
Nor I see an evident candidate for it here:
https://www.geowebcache.org/docs/current/rest/index.html

Reply:
I think this is pretty much used as a "www" folder - but I don't think there's an actual folder for it.  I think you just get the files that are need by GWC (i.e. openlayers for the demos) that are included in GWC.  

The files are here;
https://github.com/GeoWebCache/geowebcache/tree/main/geowebcache/core/src/main/resources/org/geowebcache/rest/webresources

Looking at the GWC WAR,  I don't see the files - so they are probably being accessed as a java resource.  CF. ByteStreamController

Reply:
I think this is ok in Geoserver.  The functionality is to allow geoserver-workspace requests for these files.

Reply:
(no action)